hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -82,6 +82,8 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
      * @throws NullPointerException if one of the arguments is {@code null}
      */
     @Override
+    // Suppress the warning that we shouldn't throw generic exceptions
+    @SuppressWarnings("java:S112")
```

Comment:
This code is actually wrong. `handle()` should never throw a `RuntimeException` for a user error. Instead, it should throw a `HandleException` with the error code (probably `INVALID_TRANSACTION_BODY` in this case).

Apart from that, we need to override `computeFees()` and move the code there. But this is probably something for another PR.